components/FeaturedStories/FeatureMasonry.tsx

```diff
@@ -3,64 +3,102 @@ import { Container } from '@/components/Container';
 import { Grid } from '@/components/Grid';
 import { FlexBox } from '@/components/FlexBox';
 import { EmbedMedia } from '@/components/EmbedMedia';
+import { ImageOverlay } from '@/components/ImageOverlay';
+import { Quote } from '@/components/Quote';
+import { Text } from '../Typography';
 import { getProcessedImage } from '@/utilities/getProcessedImage';
 
 
 type FeatureMasonryProps = React.HTMLAttributes<HTMLDivElement> & {
-  videoUrl?: string;
   audioUrl?: string;
+  audioBgImageSrc?: string;
+  audioBgImageFocus?: string;
+  quoteBody?: string;
+  quoteBgImageSrc?: string;
+  quoteBgImageFocus?: string;
   imageSrc1?: string;
   imageFocus1?: string;
-  imageSrc2?: string;
-  imageFocus2?: string;
-  quoteBody?: React.ReactNode;
+  imageAlt1?: string;
```

Comment:
This is an interesting naming convention. Can you tell me more about it?

Reply:
There used to be 2 images thus the 1. I should probably get rid of it 😂 Originally it's only meant to be a one-off thing in the mega feature, but I made it into a Storyblok component anyway since they could reuse it else where.